### PR TITLE
Update add and delete commands in UG

### LINKED ISSUE #160: Update add and delete command in UG

### PR DESCRIPTION
**Pull request by wujiaweijoanna**
Fixes #160

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -76,15 +76,16 @@ Format: `help`
 
 Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME e/EMAIL g/GENDER a/AGE [d/DETAIL] [t/STUDY_GROUP_TAG]…​`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
+* Gender and study group tags are **case-insensitive**
```
Comment (wkxcass) [inline docs/UserGuide.md:81]: If I'm not wrong, we would wanna make `Email` case-insensitive as well!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -127,19 +128,18 @@ Examples:
...
-* When using ranges, ensure there are **no spaces** around the hyphen (e.g., 1-5 is valid, while 1 - 5 is not).
+* Deletes the person at the specified `INDEX` or range of indices
+* The index must be within the range
+* When using ranges, ensure there are **no spaces** around the hyphen (e.g., `1-5` is valid, while `1 - 5` is not).
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
+* `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 * `delete 1 3 5-7` deletes the 1st, 3rd, and persons from the 5th to the 7th
```
Comment (wkxcass) [inline docs/UserGuide.md:144]:
```suggestion
* `delete 1 3 5-7` deletes the 1st, 3rd, 5th, 6th and 7th person in the displayed list.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -127,19 +128,18 @@ Examples:
...
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
-* When using ranges, ensure there are **no spaces** around the hyphen (e.g., 1-5 is valid, while 1 - 5 is not).
+* Deletes the person at the specified `INDEX` or range of indices
+* The index must be within the range
+* When using ranges, ensure there are **no spaces** around the hyphen (e.g., `1-5` is valid, while `1 - 5` is not).
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
+* `delete 2` deletes the 2nd person in the address book.
```
Comment (wkxcass) [inline docs/UserGuide.md:142]:
```suggestion
* `delete 2` deletes the 2nd person in the displayed list.
```

**Review by wkxcass — COMMENTED** [review]
Just a few nits! Looks good overall~

File: docs/UserGuide.md (hunk 3 above)
Comment (wujiaweijoanna) [reply docs/UserGuide.md:142]: Thx for all the comments! Nice catch :)

**Pull request merged**